ticktock-runtime/src/main/java/dev/zacsweers/ticktock/runtime/TickTockZoneRulesProvider.java

```diff
@@ -25,33 +27,48 @@
 import java.util.concurrent.ConcurrentSkipListMap;
 import java.util.function.Supplier;
 
-import static java.util.Objects.requireNonNull;
-
+/**
+ * TickTock implementation of {@link ZoneRulesProvider}. By default, this will behave identically to
+ * {@code TzdbZoneRulesProvider}. Data loading and zone IDs can be customized via {@link
+ * TickTockPlugins}.
+ */
 public final class TickTockZoneRulesProvider extends ZoneRulesProvider {
 
   private final NavigableMap<String, ZoneRules> zoneRulesById = new ConcurrentSkipListMap<>();
 
-  private final Supplier<ZoneIdsProvider> zoneIdsProvider = Suppliers.memoize(() -> {
-    Supplier<ZoneIdsProvider> callable =
-            requireNonNull(TickTockPlugins.getZoneIdsProvider(), "No ZoneIdsProvider registered!");
-    return callable.get();
-  });
+  private final Supplier<ZoneIdsProvider> zoneIdsProvider =
+      Suppliers.memoize(
+          () -> {
+            Supplier<ZoneIdsProvider> callable =
+                requireNonNull(
+                    TickTockPlugins.getZoneIdsProvider(), "No ZoneIdsProvider registered!");
```

Comment:
For defaults to work you also need the `LazyZoneDataProvider` as default here.

Reply:
Just going to remove this for now since we have the init methods that cover them. Was trying to reduce overhead before